src/main/java/persistence/entity/JdbcTemplateDialectEntityManager.java

```diff
@@ -0,0 +1,53 @@
+package persistence.entity;
+
+import jdbc.JdbcTemplate;
+import jdbc.SimpleEntityRowMapper;
+import persistence.sql.dbms.Dialect;
+import persistence.sql.dml.DeleteDMLQueryBuilder;
+import persistence.sql.dml.InsertDMLQueryBuilder;
+import persistence.sql.dml.SelectDMLQueryBuilder;
+import persistence.sql.dml.clause.WhereClause;
+import persistence.sql.dml.clause.operator.Operator;
+import persistence.sql.entitymetadata.model.EntityColumn;
+import persistence.sql.entitymetadata.model.EntityTable;
+
+public class JdbcTemplateDialectEntityManager implements EntityManager {
+    private JdbcTemplate jdbcTemplate;
+    private Dialect dialect;
```

Comment:
```suggestion
    private final JdbcTemplate jdbcTemplate;
    private final Dialect dialect;
```
저는 final 이라는 키워드를 좋아하는데요 :)
final 을 사용하는 것과 사용하지 않는 것엔 어떤 차이가 있을까요~

Reply:
인스턴스 변수에서의 final 키워드는 의도하지 않은 초기화 방지(생성자 및 변수선언시 할당만 강제) 및 불변 객체를 유지하는데 도움이 될 것 같습니다! 저도 final 과 친해져보도록 할게요 :) 감사합니다.